examples/workflows/plot_simulate_beta.py

```diff
@@ -0,0 +1,234 @@
+def add_beta_drives(net, beta_start):
+    # Distal Drive
+    weights_ampa_d1 = {'L2_basket': 0.00032, 'L2_pyramidal': 0.00008,
+                       'L5_pyramidal': 0.00004}
+    syn_delays_d1 = {'L2_basket': 0.5, 'L2_pyramidal': 0.5,
+                     'L5_pyramidal': 0.5}
+    net.add_bursty_drive(
+        'beta_dist', tstart=beta_start, tstart_std=0., tstop=beta_start + 50.,
+        burst_rate=1., burst_std=10., numspikes=2, spike_isi=10, repeats=10,
+        location='distal', weights_ampa=weights_ampa_d1,
+        synaptic_delays=syn_delays_d1, seedcore=2)
+
+    # Proximal Drive
+    weights_ampa_p1 = {'L2_basket': 0.00004, 'L2_pyramidal': 0.00002,
+                       'L5_basket': 0.00002, 'L5_pyramidal': 0.00002}
+    syn_delays_p1 = {'L2_basket': 0.1, 'L2_pyramidal': 0.1,
+                     'L5_basket': 1.0, 'L5_pyramidal': 1.0}
+
+    net.add_bursty_drive(
+        'beta_prox', tstart=beta_start, tstart_std=0., tstop=beta_start + 50.,
+        burst_rate=1., burst_std=20., numspikes=2, spike_isi=10, repeats=10,
+        location='proximal', weights_ampa=weights_ampa_p1,
+        synaptic_delays=syn_delays_p1, seedcore=8)
+
+    return net
+
+###############################################################################
+# We can now use our functions to create three distinct simulations:
+# 1) beta event only, 2) ERP only, and 3) beta event + ERP.
+beta_start, stimulus_start = 50.0, 125.0
+net_beta = net.copy()
+net_beta = add_beta_drives(net_beta, beta_start)
+
+net_erp = net.copy()
+net_erp = add_erp_drives(net_erp, stimulus_start)
+
+net_beta_erp = net_beta.copy()
+net_beta_erp = add_erp_drives(net_beta_erp, stimulus_start)
+
+###############################################################################
+# And finally we simulate. Note that the default simulation time has been
+# increased to 400 ms to observe the long time course over which beta events
+# can influence sensory input to the cortical column.
+dpls_beta = simulate_dipole(net_beta, postproc=False)
+dpls_erp = simulate_dipole(net_erp, postproc=False)
+dpls_beta_erp = simulate_dipole(net_beta_erp, postproc=False)
+
+###############################################################################
+# By inspecting the activity during the beta event, we can see that spiking
+# occurs exclusively at 50 ms, the peak of the gaussian distributed proximal
+# and distal inputs. This spiking activity leads to sustained GABAb mediated
+# inhibition of the L2 and L5 pyrmaidal cells. One effect of this inhibition
+# is an assymetric beta event with a long positive tail.
+import matplotlib.pyplot as plt
+import numpy as np
+fig, axes = plt.subplots(4, 1, sharex=True, figsize=(7, 7),
+                         constrained_layout=True)
+net_beta.cell_response.plot_spikes_hist(ax=axes[0], show=False)
+axes[0].set_title('Beta Event Generation')
+plot_dipole(dpls_beta, ax=axes[1], layer='agg', tmin=1.0, show=False)
```

Comment:
why not smooth here as well? For ease of comparison with the other graphs and real data?

Reply:
Eh I think smoothing here really obstructs your ability to see the unique features of the beta event. Also this better reflects the paper where no smoothing was used to plot the beta events (and a mixture of smoothed/unsmoothed for the ERP's)